src/plugins/legacy/medVtkView/medVtkView.cpp

```diff
@@ -223,12 +231,16 @@ QWidget* medVtkView::viewWidget()
     return d->viewWidget;
 }
 
+QMainWindow* medVtkView::mainWindow()
```

Comment:
Pourquoi?

Reply:
This is needed to get the main Window, from the medVtkView. The histogram is built on it